pulp_smash/tests/docker/api_v2/test_tags.py

```diff
@@ -96,12 +84,15 @@ def setUp(self):
         self.tags = self._get_tags()
 
     def _get_tags(self):
-        return search_units(self.cfg, self.repo, {'type_ids': ['docker_tag']})
+        return utils.search_units(
+            self.cfg,
+            self.repo, {'type_ids': ['docker_tag']}
```

Comment:
Do you mind breaking the line after the `,` so the criteria dict goes on it own line?